jac-uk/admin
# #2127 Search Improvements Help Panel

## What's included?
Added help panel for search.
Updated empty search results panel.

Test at this url: https://jac-admin-develop--pr2139-feature-2127-search-ug76l1oz.web.app/

Closes #2127
Closes #2128 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Ensure the following searches work correctly and display the correct information in the help box.

- Exercises Search
- Candidates Search
- Applications Search
- Independent Assessments Search
- Character Checks Search
- Stages (review)

See the screen recording below as an example of the Exercises Search.

https://github.com/jac-uk/admin/assets/115651787/bb033c83-57a7-4484-8715-407c2bde5087

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
The Empty search should look like this:

<img width="1204" alt="266007148-3572f595-b89c-44de-983f-84d0b61f03af" src="https://github.com/jac-uk/admin/assets/115651787/82eb82e0-3b10-4f30-9d41-d00d47c5fe24">


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
